src/functions-and-arrays.js

```diff
@@ -1,41 +1,162 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers( a, b) {
+  if (a>b){
+    return a;
+  }
+  else if (a<b){
+    return b;
+  }
+  else{
+    return a;
+  }
+}
 
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(wordArray) {
```

Comment:
this can be simplified. 
```javascript
if (!arr.length) return null;
let longestWordSoFar = arr[0];
for (let word of arr) {
  if (word.length > longestWordSoFar.length) {
    longestWordSoFar = word;
  }
}
return longestWordSoFar;
  ```